testnets/holesky/presets/sidecar-delegations-preset.env.example

```diff
@@ -0,0 +1,60 @@
+# Ethereum Node Connections + PBS URLs
+
+# Port to listen on for incoming JSON-RPC requests of the Commitments API. This
+# port should be open on your firewall in order to receive external requests!
+BOLT_SIDECAR_PORT=8017
+# Execution client API URL
+BOLT_SIDECAR_EXECUTION_API_URL="http://172.56.0.1:8545"
+# URL for the beacon client
+BOLT_SIDECAR_BEACON_API_URL="http://172.56.0.1:5052"
+# Execution client Engine API URL. This is needed for fallback block building
+# and must be a synced Geth node
+BOLT_SIDECAR_ENGINE_API_URL="http://172.56.0.1:8551"
+# The port from which the Bolt sidecar will receive Builder-API requests from the Beacon client
+BOLT_SIDECAR_CONSTRAINTS_PROXY_PORT=18550
+# URL to forward the constraints produced by the Bolt sidecar to a server
+# supporting the Constraints API, such as an MEV-Boost fork
+BOLT_SIDECAR_CONSTRAINTS_API_URL="http://bolt-mev-boost-holesky:18551"
+# The JWT secret token to authenticate calls to the engine API. It can be
+# either be a hex-encoded string or a file path to a file containing the
+# hex-encoded secret.
+BOLT_SIDECAR_ENGINE_JWT_HEX=
+# The fee recipient address for fallback blocks
+BOLT_SIDECAR_FEE_RECIPIENT=
+# Secret ECDSA key used to sign commitment messages on behalf of your validators. The public
+# key associated is an operator registered in the `BoltManager` contract and in a restaking
+# protocol.
+BOLT_SIDECAR_COMMITMENT_PRIVATE_KEY=
+# Secret BLS key to sign fallback payloads with. This can be any key. You can generate one
+# with the `bolt` CLI tool, using `bolt generate bls`.
+BOLT_SIDECAR_BUILDER_PRIVATE_KEY=
+
+# Commitments limits
+# Max number of commitments to accept per block
+BOLT_SIDECAR_MAX_COMMITMENTS_PER_SLOT=128
+# Max committed gas per slot
+BOLT_SIDECAR_MAX_COMMITTED_GAS_PER_SLOT=10_000_000
+# Min priority fee to accept for a commitment
+BOLT_SIDECAR_MIN_PRIORITY_FEE=4_000_000_000 # 4 Gwei = 4 * 10^9 wei
+
+# Chain configuration
+# Chain on which the sidecar is running
+BOLT_SIDECAR_CHAIN="holesky"
+# The slot time duration in seconds. If provided, it overrides the default for
+# the selected [chain]
+BOLT_SIDECAR_SLOT_TIME=12
```

Comment:
no reason to have this here in my opinion

Reply:
Agreed